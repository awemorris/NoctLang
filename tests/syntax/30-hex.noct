func main() {
    print(0x1);
    print(0x2);
    print(0x3);
    print(0x4);
    print(0x5);
    print(0x6);
    print(0x7);
    print(0x8);
    print(0x9);
    print(0xa);
    print(0xb);
    print(0xc);
    print(0xd);
    print(0xe);
    print(0xf);
    print(0x10);
    print(0xff);
    print(0x100);
    print(0x1000);
    print(0xf0f0);
    print(0x0f0f0);
    print(0xffffFFFF);
    print(0x100000000);
}
