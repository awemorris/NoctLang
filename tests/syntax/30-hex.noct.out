1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
255
256
4096
61680
61680
-1
0
